content/Watsonx/WatsonxData/205.md

```diff
@@ -0,0 +1,250 @@
+---
+title: '205: Ingesting Data'
+timeToComplete: 20
+updated: 2024-02-02
+---
+
+## Ingesting Data
```

Comment:
Add quiz alert tag 

<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />

Reply:
Added